kubetest/main.go

```diff
@@ -131,6 +132,7 @@ func defineFlags() *options {
 	flag.StringVar(&o.kubemarkNodes, "kubemark-nodes", "", "Number of kubemark nodes to start")
 	flag.StringVar(&o.logexporterGCSPath, "logexporter-gcs-path", "", "Path to the GCS artifacts directory to dump logs from nodes. Logexporter gets enabled if this is non-empty")
 	flag.StringVar(&o.metadataSources, "metadata-sources", "images.json", "Comma-separated list of files inside ./artifacts to merge into metadata.json")
+	flag.Var(&o.multiClusters, "multi-clusters", "If set, bring up/down multiple clusters specified. Format is [Zone1:]Cluster1[,[Zone2:]Cluster2]. Specify optional zone to bring-up cluster in specific zone")
```

Comment:
I was not thinking in this angle. Mainly because my impression of kubetest is that it is consumed only by CI jobs for now. 
I would add the error checks for now for other deployment methods.